js/handlers/expenses.js

```diff
@@ -94,3 +94,21 @@ function expenseItem (expense) {
     <time date="${expense.date}">${(new Date(expense.date)).toDateString().slice(0, -5)}</time>
   </li>`
 }
+
+export function onSettleUpClick (event) {
+  event.preventDefault()
```

Comment:
If we use a button instead of a link, probably no need to `preventDefault` either